subworkflows/local/align_cellrangermulti_idx.nf

```diff
@@ -0,0 +1,89 @@
+//
+// Include modules
+//
+include { CELLRANGER_MKGTF                  } from "../../modules/nf-core/cellranger/mkgtf/main.nf"
+include { CELLRANGER_MKREF                  } from "../../modules/nf-core/cellranger/mkref/main.nf"
+include { CELLRANGER_MKVDJREF               } from "../../modules/nf-core/cellranger/mkvdjref/main.nf"
+
+// Define workflow to subset and index a genome region fasta file
+workflow CELLRANGER_MULTI_REF {
```

Comment:
Please make sure the workflow name and filename match.

Reply:
Do you accept the new name of the file?
`cellrangermulti_ref.nf`